platform/core/src/utils/measurementFilters.ts

```diff
@@ -0,0 +1,54 @@
+export function filterTracked(trackedStudy: string, trackedSeries) {
```

Comment:
some are obvious some are not, can you add docs for non obvious ones at least? filterOr, filterUnless,